emscripten.py

```diff
@@ -2276,17 +2265,19 @@ def main(args, compiler_engine, cache, temp_files, DEBUG):
              temp_files=temp_files, DEBUG=DEBUG)
 
 
-def _main(args=None):
-  if args is None:
-    args = sys.argv[1:]
+def _main():
```

Comment:
This probably breaks when invoking `emcc`.

`emcc.py` calls `shared.Building.emscripten(...)` which does:
```python
    from emscripten import _main as call_emscripten
    settings = Settings.serialize()
    args = settings + extra_args
    cmdline = [filename + ('.o.ll' if append_ext else ''), '-o', filename + '.o.js'] + args
    with ToolchainProfiler.profile_block('emscripten.py'):
      call_emscripten(cmdline)
```

I'd suggest leaving this with the `args` argument and invoking `_main` down below as `sys.exit(_main(sys.argv[1:]))`, which secretly I like better anyway